android/quest/src/main/java/org/smartregister/fhircore/quest/ui/main/AppMainUiState.kt

```diff
@@ -47,5 +53,15 @@ fun appMainUiStateOf(
     languages = languages,
     navigationConfiguration = navigationConfiguration,
     registerCountMap = registerCountMap,
+    progressPercentage = progressPercentage,
+    isSyncUpload = isSyncUpload,
+    isSyncCompleted = syncStatus,
   )
 }
+
+enum class SyncStatus {
```

Comment:
Would it be possible to use `CurrentSyncJobStatus` directly here? Do you have any explanation as to why this enum is required?

Reply:
@ellykits 
thanks for the suggestion
looking into this.